prose.md

```diff
@@ -16,30 +16,31 @@ more than 420 words per section.
 > timing 2021-06-10: 04:01.98
 
 
-The Carpentries is a global organization of volunteers who strive to teach
+The Carpentries is a global organization of volunteers who teach
 foundational data and coding skills to researchers worldwide. What makes us
 stand out is not the fact that we use evidence-based practices in teaching, it's
-that we align all of our decisions with our core values which puts people first
-and accepts all contributions.
+that we align all of our decisions with our core values: we put people first
+and we value all contributions.
 
-Our lessons are all open-license websites built on a consistent style that
+Our lessons are distributed as open-licensed websites built on a consistent style that
 adheres to active learning principals. We use these lessons as the source
-material for the hundreds of workshops we run each year and have three distinct
+material for the hundreds of workshops we run each year. So, these sites have three distinct
 audiences:
 
-1. The certified Carpentries instructor who uses it for reference material as
-   they teach.
-2. The learner in a Carpentries workshop who uses the lesson as a reference
-   later on as they practice the materials.
-3. The educator who wants to use these open-licensed materials for their own
+1. The Carpentries Instructors who refer to these materials as
```

Comment:
I think it's good to highlight that we do certify the instructors to give a little more weight to their role. 

```suggestion
1. The certified Carpentries Instructors who refer to these materials as
```